examples/tactics/RewriteTactic.fst

```diff
@@ -0,0 +1,24 @@
+(*
+   Copyright 2008-2018 Microsoft Research
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*)
+module RewriteTactic
+
+open FStar.Tactics
+
+let lem () : Lemma (1 == 2) = admit ()
+let tau () = apply_lemma (`lem)
+
+let test _ =
```

Comment:
Could you please also add a negative testcase where, e.g., the tactic leaves a uvar and then the rewrite fails?

Reply:
Done